environment.yml

```
@@ -29,6 +29,7 @@ dependencies:
   - redis-py==5.0.1
   - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
   - pip:
+    - Scrapy==2.11.1
```

```suggestion
  - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
  - scrapy==2.11.1
  - pip:
```

Hi @Flame1190 , we generally prefer to install dependecies with conda as our first option rather than pip. Could you please test moving it under conda to ensure everything still works correctly and the code runs smoothly? Many thanks.